### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by vldtsw**

### CODE REVIEW

File: taxi/models.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        verbose_name_plural = "drivers"
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name} ({self.username})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(to=Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ForeignKey(to=settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```
Comment (Abnormaltype) [inline taxi/models.py:32]: Pls check the scheme:
https://user-images.githubusercontent.com/80070761/159295912-d02c7080-09a7-41ec-aa86-b0ae3afdd75b.png
You need create ManyToMany field:
`drivers = models.ManyToManyField(to=settings.AUTH_USER_MODEL, related_name="cars")`

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: taxi/models.py (hunk 1 above)
Comment (vldtsw) [reply taxi/models.py]: Thanks, fixed!

File: taxi/models.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        verbose_name_plural = "drivers"
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name} ({self.username})"
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(to=Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(to=settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```
Comment (Abnormaltype) [inline taxi/models.py:32]: pls add migration for this field

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

File: taxi/models.py (hunk 2 above)
Comment (vldtsw) [reply taxi/models.py]: My bad, fixed

**Review by Abnormaltype — APPROVED** [review]

**Review by IvanRamyk — APPROVED** [review]
Well done!

**Pull request closed without merging**